chebpy/core/importing.py

```diff
@@ -0,0 +1,19 @@
+import importlib
+import os
+
+def import_optional(name, envvarname, fallback=None):
+    """Attempt to import the specified module.
+    Either returns the module, or None.
+
+    See https://github.com/pandas-dev/pandas/blob/master/pandas/compat/_optional.py
+    """
+    use_module = os.environ.get('CHEBPY_USE_'+envvarname.upper(), '1')
+    if use_module.lower() in ['true', '1']:
+        try:
+            return importlib.import_module(name)
+        except ImportError:
+            pass
```

Comment:
These two lines are missed in coverage of `chebpy.core.importing`, but I think it's safe to say they don't need testing.

Should I add a ` # pragma: no cover`?

Reply:
yup, makes sense